src/main/java/io/tiledb/java/api/Array.java

```diff
@@ -370,6 +370,19 @@ public static void consolidate(
     }
   }
 
+  /**
+   * Cleans up the array, such as consolidated fragments and array metadata. Note that this will
+   * coarsen the granularity of time traveling (see docs for more information).
+   *
+   * @param arrayURI The array URI
+   * @param arrayURI The array URI
+   * @throws TileDBError A TileDB exception
+   */
+  public static void vacuum(Context ctx, String arrayURI) throws TileDBError {
```

Comment:
Can we add an overload which takes a config object? The details should highlight that the config will be used for the consolidation parameters.

Add a note to this function that the contex's config will be used.

Reply:
done